### DISCUSSION TITLE
AbortSignal in createServerFn call signature

### DISCUSSION THREAD

**Discussion by phuctm97** [TanStack/router#3069] [Ideas]
Allow to input an AbortSignal when calling a server function

Obvious use case is to call server function in route loader and pass the loader's abortController.signal to it.

**Comment by ryanagillie** [comment]
+1! Using with TanStack Query and it would be nice to be able to pass AbortControllers from the queryFn to stop in flight requests on unmount

**Comment by ryanagillie** [comment]
Someone opened an issue for this #3452
